app/assets/javascripts/services/miq_service.js

```diff
@@ -53,6 +54,14 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', funct
     $('#flash_msg_div').text("");
   };
 
+  this.miqFlashLater = function(msgObj) {
```

Comment:
why do we need this method? all it does is call the global one, can we avoid adding this?